### PR TITLE
Closes #147: Add test for `CreateMilestones` command

### LINKED ISSUE #147: `CreateMilestones` command is not covered by any tests
|    Q        |   A
|------------ | ------
| Version(s)  | 1.12.0

#### Summary

This code does not seem to be touched/verified by any existing tests:

https://github.com/laminas/automatic-releases/blob/da94ae97dae2fbbc63da7a7c82e03c362fe37d19/src/Application/Command/CreateMilestones.php#L16-L52

### PR DESCRIPTION
**Pull request by alexraputa**
|    Q          |   A
|-------------- | ------
| Documentation | no
| Bugfix        | no
| BC Break      | no
| New Feature   | no
| RFC           | no
| QA            | yes

### CODE REVIEW

File: test/unit/Application/CreateMilestonesTest.php [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+use Laminas\AutomaticReleases\Github\Api\V3\CreateMilestoneFailed;
+use Laminas\AutomaticReleases\Github\Event\Factory\LoadCurrentGithubEvent;
+use Laminas\AutomaticReleases\Github\Event\MilestoneClosedEvent;
+use Laminas\AutomaticReleases\Github\Value\RepositoryName;
+use PHPUnit\Framework\MockObject\MockObject;
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Console\Input\ArrayInput;
+use Symfony\Component\Console\Output\NullOutput;
+
+final class CreateMilestonesTest extends TestCase
```
Comment (Ocramius) [inline test/unit/Application/CreateMilestonesTest.php:22]: Please also add an `@covers` here

**Review by Ocramius — APPROVED** [review]

File: test/unit/Application/CreateMilestonesTest.php (hunk 1 above)
Comment (alexraputa) [reply test/unit/Application/CreateMilestonesTest.php:22]: Done.

Comment (Ocramius) [reply test/unit/Application/CreateMilestonesTest.php:22]: Awesome, thanks!

**Review by Ocramius — APPROVED** [review]
🚢 

Thanks @alexraputa!

**Pull request merged**